src/vm/perfmap.cpp

```diff
@@ -50,6 +51,9 @@ void PerfMap::Initialize()
         {
             s_ShowOptimizationTiers = true;
         }
+#ifndef CROSSGEN_COMPILE
```

Comment:
Nit: Have JITDUMP_SUPPORTED set to false in CROSSGEN builds so we don't need to surround every API call with these ifdefs?